src/pages/billing/models/BillingProject.ts

```diff
@@ -0,0 +1,26 @@
+import { AzureManagedAppCoordinates } from 'src/pages/billing/models/AzureManagedAppCoordinates'
+
+
+export type CloudPlatform = 'GCP' | 'Azure'
+
+export interface BillingProject {
+  cloudPlatform: CloudPlatform
+  projectName: string
+  invalidBillingAccount: boolean
+  roles: string[]
```

Comment:
Type for `roles` should be `BillingRole`.

I would also suggest just putting `BillingRole` directly in this file instead of having a separate `BillingRole.ts` file.